examples/runFasterCode.ts

```diff
@@ -71,14 +71,35 @@ const repositoryLookupName = repositoryIdentifiers.map((identifier) => {
 
 const labels = ['good first issue', 'help wanted', 'documentation'];
 const finder = new IssueFinder(logger);
+
+function convertToRecord(issues: globalThis.Map<string, Issue[]>): void {
+    const record: Record<string, Issue[]> = {};
+    issues.forEach((issuesForRepo: any, repo: any) => {
+        record[repo] = issuesForRepo;
+    });
+    const jsonFile = outputToJson(record);
+
+    return jsonFile;
+}
+
+function outputToJson(record: Record<string, Issue[]>): void {
+    const jsonResults = JSON.stringify(record, undefined, 2);
+    const data = fs.writeFileSync(outputFilePath, jsonResults);
+
+    return data;
+}
+
 finder
     .findIssues(token, labels, repositoryLookupName)
     .then((issues) => {
         let issueCount = 0;
         issues.forEach((issuesForRepo) => {
             issueCount += issuesForRepo.length;
         });
+        const directory = '\u001b[32m examples/output.json';
+        convertToRecord(issues);
         logger.info(`Found ${issueCount} issues in ${issues.size} projects\n`);
+        logger.info(`Saved issue results to: ${directory}`);
```

Comment:
It looks like the thing called "directory" is really the filename, so the variable should probably be named outpufFilename or similar.